source/index.rst

```diff
@@ -12,12 +12,18 @@ Data formats for gamma-ray astronomy
 .. image:: https://licensebuttons.net/l/by/4.0/88x31.png
     :target: https://creativecommons.org/licenses/by/4.0/
 
-A place to propose and share data format descriptions for gamma-ray astronomy.
+The *Data formats for gamma-ray astronomy* is a community-driven initiative for the definition of a common and open high-level data format for gamma-ray instruments.
 
 * Repository: https://github.com/open-gamma-ray-astro/gamma-astro-data-formats
 * Docs: https://gamma-astro-data-formats.readthedocs.io/
 * Mailing list: https://lists.nasa.gov/mailman/listinfo/open-gamma-ray-astro
 
+References
+==========
```

Comment:
Using `=` makes a `h1`, we should not have multiple `h1` head lines on the same page. Use `-` and maybe also add a 

```
Content
---------
```

Further down, since we now have the toctree under "References"

Reply:
see https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

Reply:
done